cmake/nuttx_kconfig.cmake

```diff
@@ -170,7 +186,8 @@ endfunction()
 
 function(nuttx_olddefconfig)
   execute_process(
-    COMMAND olddefconfig
+    COMMAND python3
+            ../prebuilts/tools/python/dist-packages/kconfiglib/olddefconfig.py
```

Comment:
why hard code prebuilt path in nuttx folder? all change in nuttx folder should be worked with official mainline.